<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>casingDW</class>
 <widget class="QDockWidget" name="casingDW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Initial Load</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Axial Stress:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="Sa_label">
           <property name="text">
            <string>Sa:</string>
           </property>
          </widget>
         </item>
         <item>
         <widget class="TPBrUnitInput" name="Sa" native="true">
         <property name="unitType" stdset="0">
          <enum>EPressure</enum>
         </property>
         </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Geometry</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Dimensions</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="ri_label">
           <property name="text">
            <string>r&lt;SUB&gt;i&lt;/SUB&gt;:</string>
           </property>
          </widget>
         </item>
         <item>
         <widget class="TPBrUnitInput" name="ri" native="true">
         <property name="unitType" stdset="0">
          <enum>EDimension</enum>
         </property>
         </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="re_label">
           <property name="text">
            <string>r&lt;SUB&gt;e&lt;/SUB&gt;:</string>
           </property>
          </widget>
         </item>
         <item>
         <widget class="TPBrUnitInput" name="re" native="true">
         <property name="unitType" stdset="0">
          <enum>EDimension</enum>
         </property>
         </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Mesh</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="nrad_label">
           <property name="text">
            <string>nrad:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nrad"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="initial_porder_label">
           <property name="text">
            <string>Initial p order:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="initial_porder"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>88</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="material_label">
        <property name="text">
         <string>Material:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="material_CBox">
        <item>
         <property name="text">
          <string>Linear Elastic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Von Mises</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QGroupBox" name="linearElastic_GBox">
      <property name="title">
       <string>Linear Elastic</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="E_label">
           <property name="text">
            <string>E:</string>
           </property>
          </widget>
         </item>
         <item>
         <widget class="TPBrUnitInput" name="E" native="true">
         <property name="unitType" stdset="0">
          <enum>EPressure</enum>
         </property>
         </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="nu_label">
           <property name="text">
            <string>nu:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nu"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QGroupBox" name="vonMises_GBox">
      <property name="title">
       <string>Von Mises</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>18</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TPBrUnitInput</class>
   <extends>QWidget</extends>
   <header>tpbrunitinput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
